src/System.ComponentModel.TypeConverter/src/System/Drawing/ColorConverter.cs

```diff
@@ -53,7 +53,7 @@ public override object ConvertFrom(ITypeDescriptorContext context, CultureInfo c
             string strValue = value as string;
             if (strValue != null)
             {
-                return ColorConverterCommon.ConvertFromString(strValue, culture);
+                return ColorConverterCommon.ConvertFromString(strValue, culture ?? CultureInfo.CurrentCulture);
```

Comment:
Since ColorConverterCommon is internal, I think this approach is fine, and slightly better. I'd prefer if `ConvertFromString` method had a stricter contract about what is allowed to be passed in, rather than silently use CurrentCulture instead of null.